addons/openebs/template/testgrid/k8s-docker-localpv.yaml

```diff
@@ -48,6 +53,15 @@
       namespace: openebs
       version: "__testver__"
       s3Override: "__testdist__"
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    minio_object_store_info
+    validate_read_write_object_store rwtest testfile.txt
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    minio_object_store_info
+    validate_testfile rwtest testfile.txt
```

Comment:
does this validate write after upgrade? any reason not to add validate_read_write_object_store after this?